gato/gatos/__init__.py

```diff
@@ -23,6 +24,7 @@
 import ArgentiGato
 import NormalGato
 
+importlib.reload(ReiGato)
```

Comment:
Rei above Swede and Maple below in the other PR so that hopefully they don't cause merge conflicts